test/middleware/jsonld.test.ts

```diff
@@ -72,7 +72,7 @@ describe('JSON-LD middleware', (): void => {
   });
 
   it('sets the response JSON-LD body with the dataset', async (): Promise<void> => {
-    const { response } = await makeRequest(null, null, next(null, quads));
+    const { response } = await makeRequest(undefined, undefined, next(undefined, quads));
```

Comment:
not that readable in terms of parameters, but didn't change from before

Reply:
Other option is to use an object, but is a wider refactor.

Reply:
(unrelated here I agree)